site2/docs/io-aerospike-sink.md

```diff
@@ -0,0 +1,25 @@
+---
+id: io-aerospike-sink
+title: Aerospike sink connector
+sidebar_label: Aerospike sink connector
+---
+
+The Aerospike sink connector pulls messages from Pulsar topics to Aerospike clusters.
+
+## Configuration
+
+The configuration of the Aerospike sink connector has the following properties.
+
+### Property
+
+| Name | Type|Required | Default | Description 
+|------|----------|----------|---------|-------------|
+| `seedHosts` |String| true | 3000 | The comma-separated list of one or more Aerospike cluster hosts.<br><br>Each host can be specified as a valid IP address or hostname followed by an optional port number. | 
```

Comment:
The default value for this option seems incorrect.
This is a required option. Its value is similar to `ip1:port1,ip2:port2,ip3:port3`, I think there is no need to add default values here.

Reply:
OK